docs/installation/windows.md

```diff
@@ -31,16 +31,17 @@ small ~24MB download, and boots in approximately 5s.
 
 ## Requirements
 
-Your machine must be running Windows 7.1, 8/8.1 or newer to run Docker. Windows 10 is not currently supported. To find out what version of Windows you have:
+Your machine must be running Windows 7, 8/8.1 or newer to run Docker. Windows 10 is not currently supported. To find out what version of Windows you have:
 
-1. Right click the Windows message and choose **System**. 
+1. Right click the Windows Start Menu and choose **System**.
 
     ![Which version](/installation/images/win_ver.png)
     
-    If you aren't using a supported version, you could consider upgrading your
-    operating system.
+    If you are using an supported version of Windows, you should consider upgrading your
+    operating system in order to try out Docker.
 
-2. Make sure your Windows system supports Hardware Virtualization Technology and that virtualization is enabled.
+2. Make sure your CPU supports [virtualization technology](https://en.wikipedia.org/wiki/X86_virtualization) 
+and that virtualization support is enabled in Windows.
```

Comment:
its possibly worth adding `and in your BIOS` - that's tripped up quite a few people.